lib/cmd/fhc/target.js

```diff
@@ -106,7 +106,13 @@ function target(argv, cb) {
 
       var _gotVersion = function() {
         ini.save(function(err) {
-          target.message = i18n._("Successfully targeted: ") + targ.target + (targ.user ? (" User: " + targ.user) : (" API Key: " + targ[keys.KEY_ID]));
+          var message;
+          if (targ.user) {
+            message = " User: " + targ.user;
+          } else {
+            message = " API Key: " + targ[keys.KEY_ID];
+          }
+          target.message = i18n._("Successfully targeted: " + targ.target + message);
```

Comment:
Is this functionally the same as it was before, or different?

From my reading of the old code, only `"Successfully targeted: "` was in the string passed to `i18n._()`, but now there's more in the new code. Is this okay?